### DISCUSSION TITLE
[2주차] 온라인 스터디

### DISCUSSION THREAD

**Discussion by ChaerinYu** [no-study-no-future/interview-study#20] [General]
### 진행순서
~ 20:15 각자 PR리뷰
~ 21:00 발표
~ 21:10 회고 (좋았던 점, 건의사항, 느낀점 등)

**Comment by ChaerinYu** [comment]
### 발표
1. 유채린
 * [ETC/DesignPattern] Singleton Pattern #16 
 * [Java] Garbage Collection #14 
2. 이주형
 * [Database] Index
3. 김정윤
 * [Java] 객체지향 vs 절자치향 #18

**Comment by ChaerinYu** [comment]
### 회고
* 좋았던 점
* 건의사항
* 느낀점

**Reply by ChaerinYu** [reply]
* 4주차마다 일대다 면접을 진행
  * 1~3주 동안 모든 팀원들이 발표한 내용들 기반으로 면접 스터디를 진행합니다.
  * 한 명당 15분씩 면접을 진행합니다.
* 면접 진행 전, 카훗을 통해 입풀기 시간 갖기
* 새로 오신 스터디원 동호님 환영합니다~~

**Reply by doho-ho** [reply]
- PR에 Review를 잘 작성해주신 것을 보고 흥미를 느꼈습니다. 
- PR Close할 기준을 정하면 좋을 것 같아요. (Issue 포함) : 예를 들면 더이상 해당 이슈에 대한 추가 질문, 추가할 내용이 없다면 Issue, PR 클로징
- 앞으로 잘 부탁드리겟읍니다 ^^7

**Reply by KJY97** [reply]
- 코멘트는 미리 작성하는게 좋을까요?
- 동호님 환영! 잘 부탁드립니다~

**Reply by ChaerinYu** [reply]
@Kodo92 PR Close 관련 룰은 고민 중입니다...ㅠ 다음에 얘기해봐요

**Reply by ChaerinYu** [reply]
@KJY97 코멘트는 아무 때나 ~~ 달아주시면 돼요! 발표 전에 시간을 갖긴 하니까 코멘트 못한 경우에는 그 시간 활용하시면 되어요 🙂

**Reply by leeejuhyeong** [reply]
- CS 스터디에다 면접 준비까지라니.. 너무좋아요
- 나름 발표준비해와서 깔끔했던것같아 개인적으로 만족했습니다(하핳)
- 다음 발표는 트랜잭션..
